application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksCleanupRoutine.java

```diff
@@ -0,0 +1,35 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.JDA;
+
+import org.togetherjava.tjbot.commands.Routine;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Tells the bookmarks system to clean bookmarks scheduled for removal.
```

Comment:
too implementation specific. instead, have a focus on what the class does to an user. sth like `Removes expired bookmarks. ...` and maybe add a few more details/elaborate

Reply:
Changed